inject.js

```diff
@@ -14,8 +14,14 @@ export default ({ router }) => {
       var d=document,
       g=d.createElement('script'),
       s=d.getElementsByTagName('script')[0];
-
+      
```

Comment:
Can you remove the spaces here @EloEntropy ?

Reply:
I have removed the line in the latest commit.